src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -92,6 +93,22 @@ public static Description parseDescription(String description) throws ParseExcep
         return new Description(trimmedDescription);
     }
 
+    /**
+     * Parses a {@code String priority} into a {@code Priority}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code description} is invalid.
+     */
+    public static Priority parsePriority(String priority) throws ParseException {
+        requireNonNull(priority);
+        String trimmedPriority = priority.trim();
+        if (!Priority.isValidPriority(trimmedPriority)) {
```

Comment:
This could be replaced by checkArgument() from AppUtil.java, to enhance consistency.

Reply:
Thanks for the comment! If we were to enhance consistency, wouldn't this be the best approach - considering that all of the other properties of `Task` are parsed in the same manner? (within the same `ParseUtil.java` file)

Reply:
I see, thanks!